web/packages/teleport/src/UnifiedResources/UnifiedResources.tsx

```diff
@@ -47,11 +47,6 @@ import SearchPanel from './SearchPanel';
 
 export function UnifiedResources() {
   const { clusterId, isLeafCluster } = useStickyClusterId();
-  const enabled = localStorage.areUnifiedResourcesEnabled();
-
-  if (!enabled) {
```

Comment:
I'm curious how this worked before.
If a user upgraded their cluster to v14, how will we know that they can now use unified resources? If I'm not mistaken, we never called the Unified Resources API after we set the flag?

And now the user will always see a blink of the Unified Resources loading skeleton, right? (I'm only wondering if we should change this)

Reply:
`UNIFIED_RESOURCES_NOT_SUPPORTED` is cleared from the localstorage when a session is expired so it'll recheck. 

As far as seeing the flash, I think so. However, we will remove this check entirely in 15 so then we will be supporting only 14 and nothing lower so its a temporary hack. 

After removing the change here in the code (and moving it to the fetchFunc in the unified fetch) we have the same outcome.

Reply:
>UNIFIED_RESOURCES_NOT_SUPPORTED is cleared from the localstorage when a session is expired so it'll recheck.

👍 

As we are going to backport this PR to v14, don't we want to keep the old behavior? (checking if unified resources are supported before rendering them) I think it is a bit nicer UX.

Reply:
Sure thats fine. I could have swore you recommended me to remove it in a previous review but now I can't find it. Guess it was a mistake. Added back in

Reply:
Hmm I don't recall it tbh, I only remember removing `history.replace(cfg.getNodesRoute(clusterId));` from the `resources` service :d